level3/[level3] 정윤주/java-racing-car/src/main/java/practice/Car.java

```diff
@@ -8,5 +10,18 @@ public Car(String name) {
         this.name = name;
     }
 
-    // 추가 기능 구현
+    public void move() {
+        int randomValue = Randoms.pickNumberInRange(0, 9);
+        if (randomValue >= 4) {
+            position ++;
+        }
+    }
```

Comment:
메소드 명이 구체적이지 않은 것 같아요.
단순히 move의 역할만 하고 있는 거라고 생각할 수 있는데,
실제 구현 로직을 보면 랜덤으로 움직이고 있습니다.
```suggestion
    public void randomMove() {
        int randomValue = Randoms.pickNumberInRange(0, 9);
        if (randomValue >= 4) {
            position ++;
        }
    }
```